setup/unix/traccar.xml

```diff
@@ -28,6 +28,20 @@
 
     <entry key='event.geofenceHandler'>true</entry>
 
+    <!--<entry key='mail.smtp.host'>smtp.example.com</entry>
```

Comment:
We don't need comments in release config. Better to put it in the documentation later.